Doc/library/typing.rst

```diff
@@ -572,6 +572,32 @@ These can be used as types in annotations and do not support ``[]``.
    * Every type is compatible with :data:`Any`.
    * :data:`Any` is compatible with every type.
 
+.. data:: Never
+
+   Notation for the `bottom type <https://en.wikipedia.org/wiki/Bottom_type>`_,
```

Comment:
I just noticed the Wikipedia page says `typing.NoReturn` is Python's bottom type. We live in a strange world.

Reply:
Seems like we'll be able to update that page soon :)